src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/ParameterExpander.java

```diff
@@ -521,17 +750,22 @@ public String getBuildCompletedCommand(MemoryImprint memoryImprint, TaskListener
             command = config.getGerritCmdBuildFailed();
         }
 
-        Integer verified = null;
-        Integer codeReview = null;
         Notify notifyLevel = Notify.ALL;
+        Map<String, Integer> labelVotes = new HashMap<>();
+
         if (memoryImprint.getEvent().isScorable()) {
-            verified = getMinimumVerifiedValue(memoryImprint, onlyCountBuilt);
-            codeReview = getMinimumCodeReviewValue(memoryImprint, onlyCountBuilt);
             notifyLevel = getHighestNotificationLevel(memoryImprint, onlyCountBuilt);
         }
 
-        Map<String, String> parameters = createStandardParameters(null, memoryImprint.getEvent(),
-                codeReview, verified, notifyLevel.name());
+        labelVotes.put("CODE_REVIEW", getMinimumLabelVoteValue(memoryImprint, onlyCountBuilt, CODE_REVIEW));
```

Comment:
Same question: do they really need to be handled separately?

Reply:
Nope, removed.